src/bci_build/package/__init__.py

```diff
@@ -1499,21 +1429,17 @@ def build_version(self) -> str | None:
 class ApplicationStackContainer(DevelopmentContainer):
     @property
     def _registry_prefix(self) -> str:
-        return self._image_properties.application_container_build_tag_prefix
+        if self.os_version.is_tumbleweed:
+            return "opensuse"
+        return "suse"
 
     @property
     def image_type(self) -> ImageType:
         return ImageType.APPLICATION
 
     @property
     def title(self) -> str:
-        return f"{self._image_properties.distribution_base_name} {self.pretty_name}"
-
-    @property
-    def eula(self) -> str:
-        if self.is_opensuse:
-            return "sle-bci"
-        return "sle-eula"
```

Comment:
We need this code as the otherwise all application containers switch the license to `sle-bci`.

Reply:
reverted